internal/blockprocessor/committer.go

```diff
@@ -369,21 +382,44 @@ func constructDBEntriesForDBAdminTx(tx *types.DBAdministrationTx, version *types
 			value = v
 		}
 
-		db := &worldstate.KVWithMetadata{
+		createDB := &worldstate.KVWithMetadata{
 			Key:   dbName,
```

Comment:
I am confused... this loop iterates on index definitions for existing DBs, right? because you deleted index definitions for new DBs in the previous loop. Correct?
So this essentially is an update to the value of an existing DB right? 
So the variable name is confusing - `updateDBIndex` would be more clear, IMO.

Reply:
Yes, good catch. Done.